scss/_orange-navbar.scss

```diff
@@ -27,7 +27,7 @@
   }
 
   .nav-link {
-    padding: .75rem $navbar-supra-padding-x #{map-get($spacers, 2) + $navbar-supra-padding-y} !important; // stylelint-disable-line declaration-no-important
```

Comment:
Are you sure about this modification?

It seems that the supra ba has now a height of `38px`; this screenshot shows the difference with the violet div which has a `40px` height.

<img width="1265" alt="Capture d’écran 2021-10-06 à 13 21 48" src="https://user-images.githubusercontent.com/17381666/136193416-ff1378e2-16ac-4e72-9af7-af481f1e0da7.png">

Moreover, if you don't need `$navbar-supra-padding-y` anymore, it must be removed from `_variables.scss`

Reply:
Need to double check.

Reply:
still need it, i changed paddings in an amended version of this commit